src/app/app-routing.module.ts

```diff
@@ -83,6 +84,11 @@ const routes: Routes = [
     component: VestibularesComponent,
     ...canActivate(enviarSemLogin),
   },
+  {
+    path: 'tecnologia',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/brunodiascultural/bookshelf_v1/pullRequest?prid=8812417)
```suggestion
    path: "tecnologia",
```